src/crimson/os/seastore/root_block.h

```diff
@@ -66,6 +68,11 @@ struct RootBlock : CachedExtent {
     ceph::bufferlist bl = _bl;
     bl.rebuild();
     root = *reinterpret_cast<const root_t*>(bl.front().c_str());
+    if (root.have_meta) {
```

Comment:
i am not sure i am missing something obvious. but what if a delta is intended to add meta to a root without meta?

Reply:
I thought that the whole root_t would be inclosed in the delta as indicated by the code in RootBlock::get_delta(), am I right?